### PR TITLE
Add changelog entry for 3.0.12

### PR DESCRIPTION
**Pull request by blink1073**
## References
Adds changelog entry for upcoming 3.0.12 release.

## Code changes
None

## User-facing changes
Updated changelog

## Backwards-incompatible changes
None

### CODE REVIEW

File: docs/source/getting_started/changelog.rst [hunk 1]
```diff
@@ -10,6 +10,12 @@ See the `JupyterLab
 3.0 <https://github.com/jupyterlab/jupyterlab/milestone/48?closed=1>`__
 milestone on GitHub for the full list of pull requests and issues closed.
 
+v3.0.12
+^^^^^^^
+* Remove lookbehind from ToC MD regex for Safari (`#9962 <https://github.com/jupyterlab/jupyterlab/pull/9962>`__)
+* DocRegistry FileType pattern matching doesn't work (`#9958 <https://github.com/jupyterlab/jupyterlab/pull/9958>`__)
```
Comment (jasongrout) [inline docs/source/getting_started/changelog.rst:16]:
```suggestion
* Fix DocRegistry FileType pattern matching (`#9958 <https://github.com/jupyterlab/jupyterlab/pull/9958>`__)
```

File: docs/source/getting_started/changelog.rst [hunk 2]
```diff
@@ -10,6 +10,12 @@ See the `JupyterLab
 3.0 <https://github.com/jupyterlab/jupyterlab/milestone/48?closed=1>`__
 milestone on GitHub for the full list of pull requests and issues closed.
 
+v3.0.12
+^^^^^^^
+* Remove lookbehind from ToC MD regex for Safari (`#9962 <https://github.com/jupyterlab/jupyterlab/pull/9962>`__)
```
Comment (jasongrout) [inline docs/source/getting_started/changelog.rst:15]:
```suggestion
* Fix support for Safari by changing regular expression for Table of Contents (`#9962 <https://github.com/jupyterlab/jupyterlab/pull/9962>`__)
```

**Review by jasongrout — APPROVED** [review]
Thanks!

**Comment by blink1073** [comment]
Proceeding with the release

**Comment by blink1073** [comment]
https://pypi.org/project/jupyterlab/3.0.12/

**Pull request merged**